packages/dht/src/helpers/UUID.ts

```diff
@@ -0,0 +1,30 @@
+import { v4, parse, stringify } from 'uuid'
+
+export class UUID {
```

Comment:
Would it make sense just to use instances provided by the `uuid` without wrapping to a separate class? I.e. have some helper functions (here or in those types which we need to convert to UUID)